### PR TITLE
Option to hide default options

### LINKED ISSUE #2717: Add a config to turn off base options in dropdowns
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

Some funds would prefer not to have standard issue stuff in dropdowns - let's add a config to remove.

* **What feature or behavior is this required for?**

Fund usage, esp in non-pledge contexts

* **How could we solve this issue? (Not knowing is okay!)**
Config that, when 'yes', doesn't include any of the standard issue options that are hooked up to configs. For ex, insurance options should be just [] + the config, rather than [] + the standard set + the config. 

* **Anything else?**
fund request

### PR DESCRIPTION
**Pull request by elimbaum**
I rule and have completed some work on Case Manager that's ready for review!

(brief, plain english overview of your changes here)

This pull request makes the following changes:
* adds a config to hide standard options for the requested fields
* adds tests

(If there are changes to the views, please include a screenshot so we know what to look for!)

It relates to the following issue #s: 
* Fixes #2717 
* Bumps #Y

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

**Comment by elimbaum** [comment]
Relatively straightforward changes for each of the fields — let me know if you'd prefer more tests.

File: app/helpers/patients_helper.rb [hunk 1]
```diff
@@ -37,7 +39,11 @@ def voicemail_options(current_value = nil)
         [t('dashboard.helpers.voicemail_options.no'), 'no'],
         [t('dashboard.helpers.voicemail_options.yes'), 'yes'],
     ]
-    full_set = standard_options + Config.find_or_create_by(config_key: 'voicemail').options
+    full_set = Config.find_or_create_by(config_key: 'voicemail').options
```
Comment (colinxfleming) [inline app/helpers/patients_helper.rb:42]: This also gets a little weird when the defaults aren't available, can we have this one also exempt from the hide default options rule?

**Review by colinxfleming — APPROVED** [review]
excellent work. One requested change (exempting voicemail options from this -- that's my mistake, sorry!) + a few optional changes I'll leave up to you, but this is great stuff, thank you @elimbaum !

File: app/helpers/external_pledges_helper.rb [hunk 2]
```diff
@@ -16,8 +16,13 @@ def external_pledge_source_options
       [ t('external_pledge.sources.clinic_discount'), 'Clinic discount'],
       [ t('external_pledge.sources.other_funds'), 'Other funds (see notes)']
     ]
-    external_funds.push(*standard_options)
-                  .uniq
+
+    funds = external_funds
+    unless Config.hide_standard_dropdown?
+      funds.push(*standard_options)
+    end
```
[the remark this answers is not in the document]
Comment (elimbaum) [reply app/helpers/external_pledges_helper.rb:23]: oh sick right! will change

**Comment by elimbaum** [comment]
made those changes! ready to go.

**Comment by colinxfleming** [comment]
thank you @elimbaum !

**Pull request merged**